modules/EvseManager/EvseManager.hpp

```diff
@@ -109,8 +126,24 @@ class EvseManager : public Everest::ModuleBase {
     float local_max_current_limit;
     const float EVSE_ABSOLUTE_MAX_CURRENT = 80.0;
     bool slac_enabled;
+
+    std::mutex hlc_mutex;
+
     bool hlc_enabled;
 
+    /*
```

Comment:
Ah I guess its intentional for implementing later?

Reply:
removed